srml/council/src/lib.rs

```diff
@@ -14,7 +14,246 @@
 // You should have received a copy of the GNU General Public License
 // along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
 
-//! Council system: Handles the voting in and maintenance of council members.
+//! # Council Module
+//! <!-- Original author of paragraph: @gavofyork -->
+//!
+//! ## Overview
+//!
+//! The Council module provides functionality for different voting systems to handle the voting in and maintenance of council members.
+//!
+//! - [`council::Trait`](./trait.Trait.html)
+//! - [`Call`](./enum.Call.html)
+//! - [`Module`](./struct.Module.html)
+//!
+//! ### Terminology
+//! <!-- Original author of paragraph: @gavofyork -->
+//!
+//! #### Council Proposals
+//!
+//! - **Council** Defined in the [Substrate Developer Hub Glossary](https://docs.substrate.dev/docs/glossary#section-council).
```

Comment:
Let's please not link to outside resources. The reference docs should be the ultimate source of truth.